lib/main.coffee

```diff
@@ -188,6 +188,32 @@ module.exports =
         return p
     return execPath
 
+  getProjectPath: (lintedFile) ->
```

Comment:
Note that you need to handle when that returns `null` (the file not being opened in a project).